src/main/java/aquality/appium/mobile/configuration/IConfigurationsModule.java

```diff
@@ -26,4 +29,17 @@
         return Configuration.class;
     }
 
+    /**
+     * @return class which implements {@link ISwipeConfiguration}
+     */
+    default Class<? extends ISwipeConfiguration> getSwipeConfigurationImplementation() {
+        return SwipeConfiguration.class;
+    }
+
+    /**
+     * @return class which implements {@link ITouchActions}
+     */
+    default Class<? extends ITouchActions> getTouchActionsImplementation() {
```

Comment:
the class does not belong to configuration package, so please move this method to a separate class `/mobile/actions/IActionsModule.java`